src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/InstalledPackageReference.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Versioning;
+using System;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal enum UpdateLevel { NoUpdate, Major, Minor, Patch };
+
+    /// <summary>
+    /// A class to simplify holding all of the information
+    /// about a package reference when using list
+    /// </summary>
+    internal class InstalledPackageReference
+    {
+        internal string Name { get; }
+        internal VersionRange RequestedVersion { get; set; }
```

Comment:
Mentioned it before as well. You dont need `RequestedVersion` when you already got `OriginalRequestedVersion` so remove this.